.github/workflows/count-git-contributions.yml

```diff
@@ -0,0 +1,136 @@
+name: cronjob for counting git contributions and filing a PR with results
```

Comment:
this whole script can be removed, this was to count module contributions by person to determine author order for OpenPBTA